src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -88,8 +88,10 @@ public ProCombatMoveAI(final ProAI ai) {
       clearedTerritories.add(patd.getTerritory());
     }
     territoryManager.populateEnemyAttackOptions(clearedTerritories, new ArrayList<>());
+    Set<Territory> territoriesToCheck = new HashSet<>(clearedTerritories);
+    territoriesToCheck.addAll(ProData.myUnitTerritories);
     Map<Territory, Double> territoryValueMap =
-        ProTerritoryValueUtils.findTerritoryValues(player, new ArrayList<>(), clearedTerritories);
+        ProTerritoryValueUtils.findTerritoryValues(player, new ArrayList<>(), clearedTerritories, territoriesToCheck);
```

Comment:
The `new ArrayList<>()` parameter is a smell, indicates we have an unnecessary parameter. An additional method API that accepts 3 args is typically a solution. I would not be surprised if that lead to ambiguous args in this case.